components/card.tsx

```diff
@@ -16,6 +17,7 @@ export const Card = ({
   className,
   minimal,
   fullWidth,
+  masonryCard,
```

Comment:
It would be really nice to avoid this. I think you can do what you want via CSS selectors? It's going to be annoying to have to remember to add this prop to all cards in side CardMasonry

Reply:
Yeah but then I'd have to duplicate the current cardContent items. It doesn't work out badly I think, because whenever we display cards in a masonry they'll be dynamically inserted, so only one attribute set is needed. Can always update this later if that changes